src/native/monodroid/debug.cc

```diff
@@ -603,7 +603,7 @@ Debug::enable_soft_breakpoints (void)
 void*
 xamarin::android::conn_thread (void *arg)
 {
-	abort_if_invalid_pointer_argument (arg);
+	abort_if_invalid_pointer_argument (arg, "arg");
```

Comment:
I wonder how meaningful this will be "without context": the abort messages which may be logged to the Google Play Store (594d0905db5b57f996c5fec0fbcef2a4723ea4ec) will only contain the message `Parameter 'arg' must be a valid pointer`.  *Which* "arg"?

I think the parameter should also include the method name in the message. so that a "context free" (no `adb logcat`) message at least lets us know what function we're talking about:

```
Parameter 'arg' in function 'conn_thread' must be a valid pointer.
```

Reply:
Implemented in the latest commit